modules/serverless-kafka-source-yaml.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+
+[id="serverless-kafka-source-yaml_{context}"]
+= Creating a Kafka event source using YAML
+
+You can create a Kafka event source by using YAML.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create a YAML file containing the following:
++
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1beta1
+kind: KafkaSource
+metadata:
+  name: <source-name>
+spec:
+  consumerGroup: <group-name>
+  bootstrapServers:
+  - <list-of-bootstrap-servers>
+  topics:
+  - <list-of-topics>
+  sink:
+----
++
+.Example `KafkaSource` object
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1beta1
+kind: KafkaSource
+metadata:
+  name: kafka-source
+spec:
+  consumerGroup: knative-group
+  bootstrapServers:
+  - my-cluster-kafka-bootstrap.kafka:9092
+  topics:
+  - knative-demo-topic
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+
+. Apply the YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
```

Comment:
I need to check all the verification steps for these after our 1.11.0 release anyway because atm I don't have a sensible way to test this on a cluster (without messing about with unreleased eng builds, etc). Since verification steps as a whole are optional in our doc guidelines, I will leave this one as is for now, but I will add something for it once I can test this properly.